tee-worker/client-api/src/omni/interfaces/omniExecutor/definitions.ts

```diff
@@ -130,15 +130,6 @@ export default {
         IntentNonceMismatch: "Null",
       },
     },
-    HeimaMultiSignature: {
```

Comment:
Why are we removing this one?

Reply:
We have defined [HeimaMultiSignature](https://github.com/litentry/heima/pull/3461/files/b245c68cd4ae28ea1b7310f812b9c953bb770c60#diff-25253c502d1ffbd2fc80cf50d3d662dee782f496eef992d729a102ef9e275a18R130) here.